src/content/docs/iast/install.mdx

```diff
@@ -15,85 +15,115 @@ freshnessValidatedDate: never
 To install New Relic IAST:
 
 <Steps>
-  <Step>
-    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> and click <DNT>**Set up IAST with applications**</DNT>.
+ <Step>
+    Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> and click <DNT>**Set up IAST with applications**</DNT> or <DNT>**Install**</DNT>.
 
     <img
-      title="Set up IAST with applications"
-      alt="Set up IAST with applications"
-      src="/images/iast_screenshot-full_install.webp"
-    />
-  </Step>
-
-  <Step>
-    Confirm IAST isn't running in a production environment and click <DNT>**Continue**</DNT>.  
-
-  </Step>
-
-  <Step>
-    Select the language of your application and complete the steps.
+        title="Set up IAST with applications"
+        alt="Set up IAST with applications"
+        src="/images/iast_screenshot-full_install.webp"
+    />    
+ </Step>
+ <Step>
+    Confirm that IAST isn't running in a production environment, and agree to the [service specific terms](https://newrelic.com/termsandconditions/service-specific),  and then click <DNT>**Continue**</DNT>.    
+ </Step>
+ <Step>
+    Select the  programming language of your application, update the respective APM agent, and click **Next**.
```

Comment:
What if someone doesn't know how to update their APM agent? There are different steps to update each APM agent, each of which has a separate doc on this. For example, https://docs.newrelic.com/docs/apm/agents/java-agent/installation/update-java-agent/

I think we should have link to the update docs for Go, Java, Node, and Ruby in this step. 

Also, I'd prefer "...update your APM agent..."